stable/express-gateway/Chart.yaml

```diff
@@ -9,5 +9,5 @@ maintainers:
 name: express-gateway
 sources:
 - https://github.com/expressgateway/express-gateway
-version: 1.5.0
-appVersion: 1.16.2
+version: 1.7.0
```

Comment:
Why is the chart version incrementing to `1.7.0` instead of `1.6.0`?

Reply:
@mattfarina There was another PR with 1.6 that never got merged as far as I remember; the problem is that this is an automated process and changing the commit (since you require the commit to be signed) is not possible 😱